sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/CreateChatThreadOptions.java

```diff
@@ -21,10 +21,10 @@
     /*
      * Members to be added to the chat thread.
      */
-    @JsonProperty(value = "participants", required = true)
+    @JsonProperty(value = "participants")
     private List<ChatParticipant> participants;
 
-    private String repeatabilityRequestId;
+    private String idempotencyToken;
```

Comment:
How is this property useful if it is ignored by the CreateChatThreadOptionsConverter?

Reply:
idempotencyToken is in path (not) body of the operation. See line 104 of ChatAsyncClient.java.